Applications/FileIO/GocadIO/GocadNode.h

```diff
@@ -0,0 +1,153 @@
+/**
+ * @copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#ifndef GOCADNODE_H_
+#define GOCADNODE_H_
+
+#include <bitset>
+#include <vector>
+
+#include <logog/include/logog.hpp>
+
+#include "MeshLib/Node.h"
+
+namespace MeshLib
+{
+enum class FaceIndicator : char
+{
+    U,
+    V,
+    W
```

Comment:
The names are not very descriptive. Do you have some other options?

Reply:
Renamed to FaceDirection